drivers/gpu/drm/panel/panel-ilitek-ili9881c.c

```diff
@@ -842,6 +1050,20 @@ static const struct drm_display_mode nwe080_default_mode = {
 	.height_mm 	= 170,
 };
 
+static const struct drm_display_mode cfaf7201280a0_050tx_default_mode = {
+	.clock		= 72830,
+	.hdisplay	= 720,
+	.hsync_start= 720 + 87,
+	.hsync_end	= 720 + 87 + 20, 
+	.htotal		= 720 + 87 + 20 + 87,
+	.vdisplay	= 1280,
+	.vsync_start= 1280 + 16,
+	.vsync_end	= 1280 + 16 + 8,
+	.vtotal		= 1280 + 16 + 8 + 16,
+	.width_mm	= 62,
+	.height_mm	= 1108
+};
+
```

Comment:
Indentation here has gone wrong. Kernel coding style is tabs at 8 spaces per tab.